### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN025.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#71] [General]
# https://blog.3place.cn/3place/lmyn/LMYN025.html

https://blog.3place.cn/3place/lmyn/LMYN025.html

**Comment by picklecai** [comment]
人们都说，
夜叉们并没有什么力量。
她这个弱者怎么能够
力量大如一千只象？” （1.24.2）

在这个世界里，夜叉原来比较低等。

**Comment by picklecai** [comment]
因为从前她受到神宠，
这弱者就有了力量和勇气。 （1.24.3）

这个故事的设计是：常规按等级来，非常规就是神宠的力量。
看哪里缺一难，就加一个神宠。

**Comment by picklecai** [comment]
她降生后就成长起来，
浑身洋溢着美和青春。
这绝妙的母夜叉许给了
阎浮〔233〕的儿子孙陀当夫人。 （1.24.7）

母夜叉，只是一个物种中的女性而已，既不代表力量，也不代表丑恶。

**Comment by picklecai** [comment]
浑身洋溢着美和青春的母夜叉陀吒迦，是怎么变成得又丑又坏的呢？
第一步：丈夫被仙人杀死。
第二步：带着儿子想为丈夫复仇。
第三步：儿子受到诅咒变成罗刹。
第四步：自己被诅咒而暴怒。

**Comment by picklecai** [comment]
人中英豪呀！请你不要
起什么怜悯心不把妇女杀。
为了那四个种姓的幸福，
你这王子就这样去干吧！ （1.24.15）

只要自以为正义在自己手中，就会比坏人更坏。

**Comment by picklecai** [comment]
众友的意思就是：不守达磨的坏女人，就该被杀。前面有许许多多这样的例子。你不会是第一个杀坏女人的，也不会是最后一个。

可能罗摩平时所受的教育中，对妇女是不应该起杀心的。这是一个正常教育。
